pkg/network/usm/istio.go

```diff
@@ -125,14 +125,15 @@ func (m *istioMonitor) Start() {
 	}
 
 	if m.attacher == nil {
-		log.Errorf("Istio monitoring is enabled but the attacher is nil")
+		log.Error("istio monitoring is enabled but the attacher is nil")
+		return
```

Comment:
in the `Stop` method you should be wrapping `m.attacher.Stop` with `if m.attacher != nil`